libs/elements/src/select/vl-select.element.ts

```diff
@@ -147,6 +153,18 @@ export class VlSelect extends vlFormValidationElement(BaseElementOfType(HTMLSele
         }
     }
 
+    // NOTE: UIG-2212 - HACK  we zetten de attribute disabled op de element ons zelf en niet via the vl.utils.select object van DV
+    // we nemen de css van DV maar niet alle functionaliteit dit van de disable() fn komt
```

Comment:
tekst kan beter:

bv.:

`we stellen hier zelf disabled in ipv  dit te laten doen door vl.utils.select object van DV`